cli/admin/README.md

```diff
@@ -97,27 +97,28 @@ All available commands:
 	cluster list-instances : Lists all nodes in the Pravega cluster (Controllers, Segment Stores).
 	config list : Lists all configuration set during this session.
 	config set <name=value list>: Sets one or more config values for use during this session.
+	container continuous-recover <number-of-runs> <seconds-between-runs>: Executes a local, non-invasive recovery for all SegmentContainers in the cluster during the specified duration.
```

Comment:
Was this missed documentation?

Reply:
No, the order of the commands might've been mixed around as I updated the output of the `help` command in the documentation to reflect the current list of commands.

Reply:
ok